### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by dlyr**
two things :
add # files in gitignore (emacs rocks ;)
change FindRadium and apply these changes in plugins

### CODE REVIEW

File: Plugins/FancyMesh/CMakeLists.txt [hunk 1]
```diff
@@ -1,11 +1,7 @@
 project(FancyMeshPlugin)
 
 cmake_minimum_required(VERSION 2.8.11)
-if ( NOT RADIUM_PLUGIN_OUTPUT_PATH )
-  set ( RADIUM_PLUGIN_OUTPUT_PATH "${RADIUM_ROOT}/${CMAKE_BUILD_TYPE}/Plugins")
-else()
-  message( "Output path : ${RADIUM_PLUGIN_OUTPUT_PATH}" )
-endif()
+message( "Output path FancyMesh plugin: ${RADIUM_PLUGIN_OUTPUT_PATH}" )
```
Comment (louen) [inline Plugins/FancyMesh/CMakeLists.txt]: These change are adressed in my own PR :)

**Comment by nmellado** [comment]
This branch as conflicts with master. Please fix them

**Comment by louen** [comment]
The `RADIUM_ROOT` change might break stuff in the plugins, no ?

**Comment by nmellado** [comment]
`RADIUM_ROOT` is never used directly by the plugins, in contrast to `RADIUM_PLUGIN_OUTPUT_PATH` and `RADIUM_INCLUDE_DIR`.

**Comment by louen** [comment]
Branch squashed and merged in commit  e6b1be6

**Pull request closed without merging**